.github/CHANGELOG.md

```diff
@@ -77,6 +77,12 @@
 
 ### Bug fixes
 
+* `LightningQubit` correctly decomposes state prep operations when used in the middle of a circuit.
+  [(#687)](https://github.com/PennyLaneAI/pennylane/pull/687)
+
+* `LightningQubit` correctly decomposes `qml.QFT` and `qml.GroverOperator` if wires are above a certain threshold.
```

Comment:
Can we be more explicit here for the future reference?
```suggestion
* `LightningQubit` correctly decomposes `qml.QFT` and `qml.GroverOperator` if wires are above 9 and 12 respectively.
```

Reply:
```suggestion
* `LightningQubit` correctly decomposes `qml.QFT` and `qml.GroverOperator` if `len(wires)` is above 9 and 12 respectively.
```